### DISCUSSION TITLE
Connection speed

### DISCUSSION THREAD

**Discussion by alanko0202** [directus/directus#2688] [Archive]
Hi all,
I have tested the speed of directus API using the[ demo server](https://demo.directus.io/thumper/items/customers). I would like to ask is it normal to have the TTBF around 300ms? The total request time would be on average 600ms.
Here is the [screen capture](https://prnt.sc/s7j7xu).
If I have more data, I think the speed would be slower. 

Thanks for all your efforts in this great platform.

**Comment by rijkvanzanten** [comment]
Hi there!

Our demo server runs on a pretty low-powered DigitalOcean Droplet and gets constantly hit by a lot of users, so I'm not sure if that's the most accurate representation of the true performance. That being said, there are a couple of perf improvements to be made on the API which we're actually in the middle of doing over on https://github.com/directus/api-next 🙂
